lib/katello/plugin.rb

```diff
@@ -70,6 +70,15 @@
          :engine => Katello::Engine,
          :turbolinks => false
 
+    menu :top_menu,
+         :booted_container_images,
+         :caption => N_('Booted container images'),
```

Comment:
```suggestion
         :caption => N_('Booted Container Images'),
```

Main nav and main page title are some of the few exceptions where we don't use sentence case.